Dockerfile

```diff
@@ -16,13 +16,12 @@
 #
 FROM alpine:latest as pre-build
 
-ARG APISIX_DASHBOARD_VERSION=master
+ARG APISIX_DASHBOARD_GITHASH=00000000
+
+ADD . /usr/local/apisix-dashboard
 
 RUN set -x \
-    && apk add --no-cache --virtual .builddeps git \
-    && git clone https://github.com/apache/apisix-dashboard.git -b ${APISIX_DASHBOARD_VERSION} /usr/local/apisix-dashboard \
-    && cd /usr/local/apisix-dashboard && git clean -Xdf \
-    && rm -f ./.githash && git log --pretty=format:"%h" -1 > ./.githash
+    && echo ${APISIX_DASHBOARD_GITHASH:0:8} > ./.githash
```

Comment:
why change this file?

Reply:
- The original dockerfile compilation mechanism is to compile a branch of the current warehouse. Instead of selecting the latest code in the current directory for compilation, it loses the significance of providing dockerfile files.
- The direct reason is that the configuration file with hash password is modified, and the unchanged managerapi code in the master branch cannot pass the CI test.

Reply:
hi @bzp2010 
should we also update the dockerfile in repo apisix-docker?

Reply:
#### Changes
- Modify dockerfile to compile with local branch code.
- Instead of getting githash from clone's code, pass in githash variable through GitHub actions.

Reply:
> should we also update the dockerfile in repo apisix-docker?

In fact, I don't think we need to update the `Dockerfile`. The code in that repo is mainly used to compile and upload to the registry, while the code in the current warehouse is more used for development purposes.

What do you think? Thx.